app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1574,11 +1590,9 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope, DaxDialogLi
             }
         }
 
-        fun cancelAllAnimations() {
+        fun cancelTrackersAnimation() {
             animatorHelper.cancelAnimations()
-            animationContainer.alpha = 0f
-            clearTextButton.alpha = 1f
-            omnibarTextInput.alpha = 1f
+            viewModel.stopShowingEmptyGrade()
```

Comment:
`viewModel.stopShowingEmptyGrade()` is also called from `onAnimationFinished`. Why should we call it also from here? From what I see, `cancelAnimations` will call `onAnimationFinished`. If animation is running, finishing the animation will call `onAnimationFinished`. If animation is not running, it means that animation finished and `onAnimationFinished` was already called.

is there something else?

Reply:
Good catch! I was experimenting with removing the call to the listener on `cancelAnimations` to avoid multiple state changes but ended up adding it in `stopShowingEmptyGrade` and then forgot to put everything back to how it was.